src/main/kotlin/uk/gov/justice/hmpps/offendersearch/services/PhraseQueryBuilder.kt

```diff
@@ -82,8 +82,8 @@ private fun simpleTermAndQuery(phrase: String): QueryBuilder =
         .field("firstName", 10f)
         .field("surname", 10f)
         .field("middleNames", 8f)
-        .field("offenderAliases.firstName", 8f)
-        .field("offenderAliases.surname", 8f)
+        .field("offenderAliases.firstName", 1.5f)
```

Comment:
As alias now exists in a nested document we have to alter the boost accordingly - value derived from experimentation.